source/includes/_010-introduction.md

```diff
@@ -8,7 +8,7 @@ All APIs SHOULD be accessed from `https://go-server-url:8154/go/api`. All data S
 
 ## Current version
 
-By default, all requests receive **Version 1** of the API. It is recommended to explicitly request this version via the `Accept` header.
+Since v15.2.0,  all requests receive **Version 1** of the API. It is recommended to explicitly request this version via the `Accept` header.
```

Comment:
Not all APIs have this header — Users may not be familiar with what APIs were introduced and when. I'd think rephrasing it to something along the lines of "Please check if the API request requires an `Accept` header with the api version" is probably better. WDYT?